### PR TITLE
[WIP] Allow NCOP questions to be imported

### PR DESCRIPTION
**Pull request by deanmalan**
**Remember to run migrations after deploy!**

I imported a few NCOP questions with the scraper and it worked fine. NCOP and NA questions seem to have the same structure. 

[Pivotal](https://www.pivotaltracker.com/story/show/172472649)

pa.org.za/question will redirect to pmg.org.za/question_replies/

Questions on profile page on desktop:

Questions on profile page on mobile:

### CODE REVIEW

File: pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py [hunk 1]
```diff
@@ -209,7 +209,8 @@ def handle_api_question_and_reply(self, data):
                     data['url'])
             return
         house = {
-            'National Assembly': 'N'
+            'National Assembly': 'N',
+            'National Council of Provinces': 'C'
         }.get(data['house']['name'])
         if not house:
             print "Skipping {} because the house {} is not supported.".format(
```
Comment (jbothma) [inline pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py]: it feels like time to factor out the logic in handle_api_question_and_reply - it's a loooooooong method and does a whole bunch of parsing/translation between pmg and pombola data formats

File: pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py [hunk 2]
```diff
@@ -209,7 +209,8 @@ def handle_api_question_and_reply(self, data):
                     data['url'])
             return
         house = {
-            'National Assembly': 'N'
+            'National Assembly': 'N',
+            'National Council of Provinces': 'C'
```
Comment (jbothma) [inline pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py]: what in pombola knows what to do with C? is that tested somewhere (either that it handles arbitrary house codes, or that it handles this particular one

Comment (deanmalan) [reply pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py]: https://github.com/OpenUpSA/pombola/blob/master/pombola/za_hansard/models.py#L229-L232

Comment (deanmalan) [reply pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py]: I searched through all of the occurrences of `.house` in the codebase and there weren't any on question objects.

**Pull request closed without merging**